dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -51,8 +76,16 @@ rocommunity6 {{ snmp_rocommunity }}
 
 #  Note that setting these values here, results in the corresponding MIB objects being 'read-only'
 #  See snmpd.conf(5) for more details
-sysLocation    {{ snmp_location }}
+{% if SNMP is defined and SNMP.LOCATION is defined %}
+sysLocation    {{ SNMP.LOCATION.Location }}
+{% else %}
+sysLocation    public
+{% endif %}
+{% if SNMP is defined and SNMP.CONTACT is defined %}
+sysContact     {{ SNMP.CONTACT.keys()[0] }} {{ SNMP.CONTACT.values()[0] }}
```

Comment:
Why are key and value from index 0 used? Can there be multiple CONTACTS and only first is selected if there are multiple contacts?

Reply:
There can only be one sysContact from what I can see here: 
https://tools.ietf.org/html/rfc1213

          sysContact OBJECT-TYPE
              SYNTAX  DisplayString (SIZE (0..255))
              ACCESS  read-write
              STATUS  mandatory
              DESCRIPTION
                      "The textual identification of the contact person
                      for this managed node, together with information
                      on how to contact this person."
              ::= { system 4 }

Plus there is a oid for this: 
trvanduy@netdev1-westus2-v2:~$ snmpwalk -v 2c -c <comm> 10.3.147.239 .1.3.6.1.2.1.1.4
Unlinked OID in IPATM-IPMC-MIB: marsMIB ::= { mib-2 57 }
Undefined identifier: mib-2 near line 18 of /usr/share/mibs/ietf/IPATM-IPMC-MIB
SNMPv2-MIB::sysContact.0 = STRING: joe joe@contoso.com
So that is why I have it marked to only get one.